ravelin.js

```diff
@@ -1092,13 +1092,26 @@
     var rsaResultB64 = rsaEncrypt(this.rsaKey, aesResult.aesKeyB64, aesResult.ivB64);
 
     return JSON.stringify({
+      methodType: 'paymentMethodCipher',
       cardCiphertext: aesResult.ciphertextB64,
       aesKeyCiphertext: rsaResultB64,
       algorithm: "RSA_WITH_AES_256_GCM",
       ravelinjsVersion: version,
     });
   };
 
+  /**
+   * encryptAsObject takes the encrypted card details and prepares them to be sent
+   * to Ravelin, with the resulting payload returned as an object.
+   *
+   * @param {Object} details An object containing properties pan, month, year,
+   *                         month, and nameOnCard (optional).
+   * @return {Object} The encrypted payload to be sent to Ravelin.
+   */
+  RavelinJS.prototype.encryptAsObject = function(details) {
+    return JSON.parse(this.encrypt(details));
```

Comment:
I think we should rename the existing `encrypt` to `encryptAsObject` and remove its call to JSON.stringify, then re-implement `encrypt` as `return JSON.stringify(encryptAsObject(details));`